src/JWT.js

```diff
@@ -30,11 +30,14 @@ export function verifyJWT ({registry, address}, jwt, callbackUrl = null) {
           return reject(new Error('JWT has expired'))
         }
         if (payload.aud) {
-          if (payload.aud.match(/^0x[0-9a-fA-F]+$/)) {
+          if (payload.aud.match(/^0x[0-9a-fA-F]+$/) || isMNID(payload.aud)) {
             if (!address) {
               return reject(new Error('JWT audience is required but your app address has not been configured'))
             }
-            if (payload.aud !== address) {
+
+            const addressHex = isMNID(address) ? decode(address).address : address
+            const audHex = isMNID(payload.aud) ? decode(payload.aud).address : payload.aud
+            if (audHex !== addressHex) {
```

Comment:
just making sure both addresses are in the same form for comparison, so not assuming the aud or address is MNID encoded